pkg/mpc-examples/test/03-EBURebalancer.test.ts

```diff
@@ -0,0 +1,175 @@
+import { assert, expect } from 'chai';
+import { ethers } from 'hardhat';
+import { bn, fp } from '@orbcollective/shared-dependencies/numbers';
+import { Contract } from '@ethersproject/contracts';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/dist/src/signer-with-address';
+import { getBalancerContractArtifact } from '@balancer-labs/v2-deployments';
+import * as expectEvent from '@orbcollective/shared-dependencies/expectEvent';
+import { setupEnvironment, TokenList, pickTokenAddresses } from '@orbcollective/shared-dependencies';
+import { toNormalizedWeights } from '@balancer-labs/balancer-js';
+
+export const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000';
+
+let deployer: SignerWithAddress;
+let mpcFactory: Contract;
+let ebuRebalancer: Contract;
+let tokenAddresses: string[];
+
+const initialWeights = toNormalizedWeights([fp(33.34), fp(33.33), fp(33.33)]);
+const swapFeePercentage = bn(0.3e16);
+
+async function deployBalancerContract(
+  task: string,
+  contractName: string,
+  deployer: SignerWithAddress,
+  args: unknown[]
+): Promise<Contract> {
+  const artifact = await getBalancerContractArtifact(task, contractName);
+  const factory = new ethers.ContractFactory(artifact.abi, artifact.bytecode, deployer);
+  const contract = await factory.deploy(...args);
+  return contract;
+}
+
+async function deployController(deployer: SignerWithAddress): Promise<Contract> {
+  const newPoolParams = {
+    name: 'MyTestPool',
+    symbol: 'MTP',
+    tokens: tokenAddresses,
+    normalizedWeights: initialWeights,
+    // assetManagers: [ZERO_ADDRESS, ZERO_ADDRESS, ZERO_ADDRESS],
+    swapFeePercentage: swapFeePercentage,
+    swapEnabledOnStart: true,
```

Comment:
might want to set to `false`, can discuss

Reply:
I removed and hardcoded to false within the EBURebalancerControllerFactory